dummy/src/toga_dummy/window.py

```diff
@@ -109,3 +109,15 @@ def set_full_screen(self, is_full_screen):
     @not_required
     def toga_on_close(self):
         self._action("handle Window on_close")
+
+    @not_required_on("mobile", "web")
+    def get_current_screen(self):
+        self._get_value("screen")
+
+    # @not_required_on("mobile", "web")
+    # def set_current_screen(self, app_screen):
+    #     self._set_value("screen", app_screen)
+
+    @not_required_on("mobile", "web")
+    def get_primary_screen(self):
+        return self.interface._app.screens[0]
```

Comment:
This should be a red flag - if you can provide a fully dummy implement using APIs at the interface layer... why isn't it implemented at the interface layer?

The only reason I can see that it couldn't be is that GTK and Winforms have separate "get primary display" APIs, so `screen[0]` isn't *necessarily* the primary... but that's something we can fix by *making* `screen[0]` the primary as part of the implementation.

Reply:
Now `App.screens[0]` will always be the primary display.